### ISSUE TITLE
2.3.0 bad generated upstart conf file

### ISSUE THREAD

**Issue by AntonIvanov87** [playframework/playframework#2968]
I use sbt.packager to create deb package for my play project.
After upgrading from 2.2.3 to 2.3.0 I noticed that generated upstart conf file (/etc/init/{app_name}.conf) is no longer valid.
```
$ sudo service app_name start
start: Unknown job: app_name
```
This error is because of following lines in upstart conf file:
```
...
start on started (networking)
...
stop on stopping (networking)
```
I think they should look like this:
```
...
start on (started networking)
...
stop on (stopping networking)
```
After correcting upstart conf file manually it works ok.
Here is extract from my build.sbt:
```
import com.typesafe.sbt.packager.Keys._
...
packageArchetype.java_server

maintainer := "Anton Ivanov <an.ivanov@hh.ru>"

packageSummary := "package summary"

packageDescription := "package description"

debianPackageDependencies in Debian ++= Seq("oracle-java7-jre")
```

**Comment by benmccann** [comment]
It sounds like this bug was fixed in native packager 0.7.2, so I'm closing this since Play 2.3.8 is using native packager 0.7.4

**Comment by giabao** [comment]
I still can not use upstart for my play 2.4.6 app:
+ play 2.4.6
+ enablePlugins(JDebPackaging) - build from CentOS 7
+ /etc/default/hocthuc-blue  already contains:
```
JAVA_OPTS="-Dapplication.home=/usr/share/hocthuc-blue -Dpidfile.path=/var/run/hocthuc-blue/play.pid $JAVA_OPTS"
PIDFILE="/var/run/hocthuc-blue/play.pid"
```
+ still error:
```
# tail /var/log/upstart/hocthuc-blue.log
Oops, cannot start the server.
java.io.FileNotFoundException: /usr/share/hocthuc-blue/RUNNING_PID (Permission denied)
        at java.io.FileOutputStream.open0(Native Method)
        at java.io.FileOutputStream.open(FileOutputStream.java:270)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
        at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
        at play.core.server.ProdServerStart$.createPidFile(ProdServerStart.scala:131)
        at play.core.server.ProdServerStart$.start(ProdServerStart.scala:45)
        at play.core.server.ProdServerStart$.main(ProdServerStart.scala:27)
        at play.core.server.ProdServerStart.main(ProdServerStart.scala)
```

So I need [use SystemV](http://www.scala-sbt.org/sbt-native-packager/archetypes/java_server/my-first-project.html?highlight=systemv#debian-deb)
